test/nuts/trackingCommands/customLabels.nut.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import * as fs from 'fs';
+import { expect } from 'chai';
+
+import { execCmd, TestSession } from '@salesforce/cli-plugins-testkit';
+import { AuthInfo, Connection } from '@salesforce/core';
+import { PushResponse } from '../../../src/formatters/source/pushResultFormatter';
+import { PullResponse } from '../../../src/formatters/source/pullFormatter';
+
+let session: TestSession;
+describe('CustomLabel source tracking', () => {
+  before(async () => {
+    session = await TestSession.create({
+      project: {
+        gitClone: 'https://github.com/WillieRuemmele/sfdx-delete-customlabel',
```

Comment:
I'd prefer this be in the repo, so it all comes in the initial git clone.  There's other NUT repos that are.